packages/wxt/src/core/resolve-config.ts

```diff
@@ -156,6 +157,10 @@ export async function resolveConfig(
   }
 
   const userModules = await resolveWxtUserModules(
+    path.resolve(
+      root,
+      inlineConfig.configFile ? path.dirname(inlineConfig.configFile) : '.',
+    ),
```

Comment:
Modules should always go in `<root>/modules`, where the config file is placed doesn't effect this.

```suggestion
    root,
```